BUILD_LINUX.md

```diff
@@ -93,15 +93,16 @@ git checkout master
 
 ### Using a custom Qt build
 
-Qt binaries are only provided for Ubuntu. In order to build on other distributions, a Qt5 install needs to be provided as follows:
+Qt binaries are only provided for Ubuntu. In order to build on other distributions, a Qt5 install
+ needs to be provided by setting the `VIRCADIA_QT_PATH` environment variable to a directory containing
+ a Qt install.
 
-* Set `VIRCADIA_USE_PREBUILT_QT=1`
-* Set `VIRCADIA_USE_QT_VERSION` to the Qt version (defaults to `5.12.3`)
-* Set `HIFI_QT_BASE=/path/to/qt`
+ ### Using the system's Qt
 
-Qt must be installed in `$HIFI_QT_BASE/$VIRCADIA_USE_QT_VERSION/qt5-install`.
+ The system's Qt can be used, if the development packages are installed, by setting the
+ `VIRCADIA_USE_SYSTEM_QT` environment variable. The minimum recommended version is Qt 5.15.0.
```

Comment:
We can currently still build against a patched Qt 5.12. The patch fixes a crash. You could build without the patch, it just wouldn't be reliable.

5.15 is the first version that comes with that fix already applied. Once we fully switch to 5.15 and drop compatibility with 5.12, we'll make it a hard requirement, since fixing some things 5.15 issues warnings about breaks compatibility with 5.12 anyway.

Re: quazip interesting, I've not run into that one during my testing. I'll take a look. But the `_env` thing is a mechanism to pass around environment variables to vcpkg. vcpkg works in a clean environment, so a hack was needed to circumvent that.

Reply:
Yeah. The quazip thing only happens if I try to build without vircadia-builder. I have not found any special logic inside of vircadia-builder that might fix that problem though.